### ISSUE TITLE
Please rename `DownstreamExpirationTime` to `TransactionMaxLifetime`

### ISSUE THREAD

**Issue by DDvO** [siemens/LightweightCmpRa#8]
This is a follow-up on #7, as discussed.

Please also tweak the description in `README.md`, as follows:

```
The **`TransactionMaxLifetime` object**
optionally specifies the maximum lifetime of CMP transactions.
The Lightweight CPM RA persists the message exchange state of each transaction
until its regular or erroneous termination or until its age reaches the
given number of seconds.
By default, or if the value 0 is given, transaction lifetime is not restricted.
Restricting transaction lifetime avoids blocking RA resources indefinitely
for instance when an expected subsequent request message by the client is lost
or the client terminates during a transaction without the RA knowing.
```

**Comment by Akretsch** [comment]
fixed in  03fcf34

**Comment by DDvO** [comment]
Thanks.

Yet I just found for the related CMP RA Component in https://code.siemens.com/product-pki/ct_rda/CmpRaComponent/-/blob/java-client/src/main/java/com/siemens/pki/cmpracomponent/persistency/PersistencyContext.java
something that looks like a leftover:

    public void updateDownstreamExpirationTime(final Date downstreamExpirationTime) {
        // only downstream can expire
        this.expirationTime = downstreamExpirationTime;
    }
